dev/com.ibm.ws.security.client_fat/fat/src/com/ibm/ws/security/client/fat/JavaColonInjectionsTest.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ * Copyright (c) 2015, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.security.client.fat;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ProgramOutput;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyClient;
+import componenttest.topology.impl.LibertyClientFactory;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+@SkipForRepeat(SkipForRepeat.EE9_FEATURES) // TODO jakartaMail-1.6 is not yet available.
```

Comment:
This comment should say mail-2.0.  It might be available now.

Reply:
I just looked and it is edition = beta so it would appears available.

Reply:
Removed the skip for repeat as mail-2.0 is available. I also had to update `wlp-jakartaee-transform/rules/jakarta-renames.properties` (rename javax.mail to jakarta.mail) and `com.ibm.ws.security.client_fat/bnd.bnd`  (add EE9 tested features) for this change.